client/handle_http_test.go

```diff
@@ -389,41 +386,36 @@ func generateFileTestScitoken() (string, error) {
 	if issuerUrl == "" { // if empty, then error
 		return "", errors.New("Failed to create token: Invalid iss, Server_ExternalWebUrl is empty")
 	}
-	jti_bytes := make([]byte, 16)
-	if _, err := rand.Read(jti_bytes); err != nil {
-		return "", err
-	}
-	jti := base64.RawURLEncoding.EncodeToString(jti_bytes)
-
-	tok, err := jwt.NewBuilder().
-		Claim("scope", "storage.read:/ storage.modify:/").
-		Claim("wlcg.ver", "1.0").
-		JwtID(jti).
-		Issuer(issuerUrl).
-		Audience([]string{param.Origin_Url.GetString()}).
-		Subject("origin").
-		Expiration(time.Now().Add(time.Minute)).
-		IssuedAt(time.Now()).
-		Build()
+
+	scopes := []token_scopes.TokenScope{}
+	readScope, err := token_scopes.Storage_Read.Path("/")
 	if err != nil {
-		return "", err
+		return "", errors.Wrap(err, "failed to create 'read' scope for file test token:")
 	}
-
-	key, err := config.GetIssuerPrivateJWK()
+	scopes = append(scopes, readScope)
+	modScope, err := token_scopes.Storage_Modify.Path("/")
 	if err != nil {
-		return "", errors.Wrap(err, "Failed to load server's issuer key")
+		return "", errors.Wrap(err, "failed to create 'modify' scope for file test token:")
 	}
-
-	if err := jwk.AssignKeyID(key); err != nil {
-		return "", errors.Wrap(err, "Failed to assign kid to the token")
+	scopes = append(scopes, modScope)
+
+	fTestTokenCfg := utils.TokenConfig{
+		TokenProfile: utils.WLCG,
+		Lifetime:     time.Minute,
+		Issuer:       issuerUrl,
+		Audience:     []string{param.Origin_Url.GetString()},
+		Version:      "1.0",
+		Subject:      "origin",
+		Claims:       map[string]string{"scope": token_scopes.GetScopeString(scopes)},
```

Comment:
For all the scope claim, `TokenConfig` has two built-in functions to handle it, one is `AddScopes`, which is to add an array of our pre-defined token scopes in`token_scopes.TokenScope`, the other is `AddRawScope`, which is to add any other scopes we didn't define. I'd suggest we use those two functions wherever possible to be consistent.